runtime/src/staking.rs

```diff
@@ -1506,39 +1558,21 @@ impl<T: Trait> Module<T> {
     }
 
     /// Does the given account id have compliance status `Active`
-    fn is_eligible_to_validate(account_id: &T::AccountId) -> bool {
-        let validator = Self::permissioned_validators(account_id);
-        validator.compliance == Compliance::Active
-    }
-
-    /// Non-deterministic method that checks KYC status of each validator and persists
-    /// any changes to compliance status.
-    fn refresh_compliance_statuses() {
```

Comment:
See comment above, but I think we still need this function?

Reply:
Added this back. On each new session, `refresh_compliance_statuses()` is called which calls `is_validator_compliant()` to perform KYB check.